modules/registry-configuring-storage-vsphere.adoc

```diff
@@ -47,30 +43,7 @@ $ oc get pod -n openshift-image-registry
 +
 [NOTE]
 =====
-* If the storage type is `emptyDIR`, the replica number cannot be greater than `1`.
-* If the storage type is `NFS`, you must enable the `no_wdelay` and `root_squash` mount options. For example:
-+
-[source,terminal]
-----
-# cat /etc/exports
-----
-+
-.Example output
-[source,terminal]
-----
-/mnt/data *(rw,sync,no_wdelay,root_squash,insecure,fsid=0)
-----
-+
-[source,terminal]
-----
-sh-4.2# exportfs -rv
-----
-+
-.Example output
-[source,terminal]
-----
-exporting *:/mnt/data
-----
+If the storage type is `emptyDIR`, the replica number cannot be greater than `1`.
```

Comment:
same as baremetal

Reply:
This seems correct to me.

Reply:
Will leave as is, as noted in my previous comment.